test/kitchen/tasks/kitchen.py

```diff
@@ -0,0 +1,156 @@
+from invoke import task
+from invoke.exceptions import Exit
+
+import glob
+import json
+import os.path
+import re
+
+@task(iterable=['platlist'])
+def genconfig(
+    ctx,
+    platform=None,
+    provider=None,
+    osversions="all",
+    testfiles=None,
+    uservars=None,
+    platformfile="platforms.json",
+    platlist=None
+):
+    """
+    Create a kitchen config
+    """
+    if not platform and not platlist:
+        print("Must supply a platform to configure\n")
+        raise Exit(1)
+
+    if not testfiles:
+        print("Must supply one or more testfiles to include\n")
+        raise Exit(1)
+
+    if platlist and (platform or provider):
+        print("Can specify either a list of specific OS images OR a platform and provider, but not both\n")
+        raise Exit(1)
```

Comment:
nit: [`Exit` has a ` message` field](http://docs.pyinvoke.org/en/stable/api/exceptions.html#invoke.exceptions.Exit). I think the proper way of doing this is passing the error message to the `Exit` exception:
```suggestion
        raise Exit(message="Can specify either a list of specific OS images OR a platform and provider, but not both\n", code=1)
```